_source/user-guide/integrations/endpoints.md

```diff
@@ -13,23 +13,30 @@ contributors:
 Notification endpoints can receive notifications when an alert is triggered,
 when a user shares a Kibana object,
 or when Logz.io finds new insights in your logs.
+
+
 Logz.io supports sending notifications to
-BigPanda, Datadog, PagerDuty, Slack, VictorOps, or your own custom endpoint.
+Opsgenie, BigPanda, Datadog, PagerDuty, Slack, VictorOps, or your own custom endpoint.
+
+![Notification endpoints](https://dytvr9ot2sszz.cloudfront.net/logz-docs/notification-endpoints/notification-endpoints.png)
```

Comment:
![image](https://user-images.githubusercontent.com/6845404/76897538-9c054780-689c-11ea-847b-a611cf0550f9.png)

This screen shot takes up most of the vertical space. Cutting it to the second row in the table should do the trick.

Reply:
Not sure what "cutting it to the second row" means. It is supposed to illustrate the page and also show the icons of the integrations we support "OOTB". I can rethink it.

Reply:
It means to cut the image vertically, and show the first two rows of the table only. That should cut the vertical height by half and improve the flow of the page.